<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="num1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okrdel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>//</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="umn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mod1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="skobka1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="skobka2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>390</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepen">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x^y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kvadr">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x^2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kub">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x^3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kor">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>√x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ykor">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>y√x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cos1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="yandexgolos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ask yandex.ru</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ravno">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ceil1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Округлить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tan1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>П</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EEE">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>__________________________________________________________________________________________________</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>остаток
от деления</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sbros">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="result1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="delete2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mminus">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>M-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mr">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mplus">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>M+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="probel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>kvadr</zorder>
   <zorder>num1</zorder>
   <zorder>okrdel</zorder>
   <zorder>num2</zorder>
   <zorder>num3</zorder>
   <zorder>umn</zorder>
   <zorder>del1</zorder>
   <zorder>num4</zorder>
   <zorder>num5</zorder>
   <zorder>num6</zorder>
   <zorder>mod1</zorder>
   <zorder>skobka1</zorder>
   <zorder>skobka2</zorder>
   <zorder>num7</zorder>
   <zorder>num8</zorder>
   <zorder>num9</zorder>
   <zorder>minus</zorder>
   <zorder>plus1</zorder>
   <zorder>stepen</zorder>
   <zorder>kub</zorder>
   <zorder>kor</zorder>
   <zorder>ykor</zorder>
   <zorder>cos1</zorder>
   <zorder>yandexgolos</zorder>
   <zorder>ravno</zorder>
   <zorder>sin1</zorder>
   <zorder>ceil1</zorder>
   <zorder>tan1</zorder>
   <zorder>pi</zorder>
   <zorder>EEE</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>sbros</zorder>
   <zorder>num0</zorder>
   <zorder>result1</zorder>
   <zorder>delete2</zorder>
   <zorder>Mminus</zorder>
   <zorder>Mr</zorder>
   <zorder>Mplus</zorder>
   <zorder>probel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
